charts/rancher-logging/rancher-logging/100.0.0+up3.12.0/templates/clusterrole.yaml

```diff
@@ -82,6 +82,18 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
Whats the new job stuff for ?

Reply:
looks like they added a feature where it will find abandoned PVs used for fluentd caching and drain them https://github.com/banzaicloud/logging-operator/blob/598c40b6a846c13e3a002d1c7326732829a27ad1/docs/volume-drainer.md